src/main/java/aeneas/views/BuildSelectLevelView.java

```diff
@@ -101,6 +108,57 @@ public void initialize(URL location, ResourceBundle resources) {
 
     JFXDepthManager.setDepth(fileList, 1);
 
+    refresh();
   }
 
+  public void refresh() {
+    fileList.getItems().clear();
+    
+    fileList.getItems().add(createNewLevelLabel);
+    Path defaultLevelPath = Paths.get(System.getenv("HOME"), ".aeneas-kabasuji");
```

Comment:
Yeah, if we settle for only having numbered levels (where >15 means custom) then I think we can use the same level index for the build select level as well